### ISSUE TITLE
how to run the project that under packages? eg  mini-browser

### ISSUE THREAD

**Issue by xinsuinizhuan** [theia-ide/theia#5605]

**Comment by vince-fugnitto** [comment]
I don't quite understand, do you mean how to build an individual package like `mini-browser`?
If you are running the example Theia application under `examples-browser` then you are already running the mini-browser.

**Comment by vince-fugnitto** [comment]
Also, I'm not sure if you know or not, but we have another forum where other users ask questions and get help :) https://spectrum.chat/theia?tab=posts

**Comment by xinsuinizhuan** [comment]
sorry，i make you puzzle. I only mean how to run theia with minibrower,like below?
<img width="1552" alt="53496061-865e8d80-3aa1-11e9-80bd-8f9dbe78f27d" src="https://user-images.githubusercontent.com/40679769/60320668-01cb3f80-99ad-11e9-8cf2-576c5d70d954.png">

**Comment by kittaakos** [comment]
> sorry，i make you puzzle. I only mean how to run theia with minibrower,like below?

tldr;
Use: HTTPS instead of HTTP.

Longer:
If you create an `iframe` and try the URL you provided
```html
<iframe src="http://perdu.com">
```

you will get the following error:
```
Mixed Content: The page at 'https://a0674502-6601-45ed-b78f-8b329b1bf134.ws-eu0.gitpod.io/#/workspace/theia' was loaded over HTTPS, but requested an insecure resource 'http://perdu.com/'. This request has been blocked; the content must be served over HTTPS.
```

![Screen Shot 2019-06-28 at 09 23 37](https://user-images.githubusercontent.com/1405703/60324968-6e7f1380-9986-11e9-915f-1e624d019068.jpg)

I am closing this. Feel free to reopen.
